### PR TITLE
Document hot standby for replicated event sourcing

### PR DESCRIPTION
**Pull request by johanandren**
Refs #29319

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/replicated-eventsourcing.md [hunk 1]
```diff
@@ -287,6 +287,12 @@ The "event publishing" feature publishes each event to the local system event bu
 the @apidoc[ShardingDirectReplication] actor subscribes to these events and forwards them to the replicas allowing them
 to fast forward the stream of events for the origin replica. (With additional potential future support in journals for fast forwarding [#29311](https://github.com/akka/akka/issues/29311)). 
 
+## Hot Standby
+
+If all writes occur to one replica the other replicas are not started there might be many replicated events to catch up with when they are later started. Therefore it can be good to activate all replicaswhen there is some activity.
```
Comment (chbatey) [inline akka-docs/src/main/paradox/typed/replicated-eventsourcing.md:292]: missing space replicaswhen

File: akka-docs/src/main/paradox/typed/replicated-eventsourcing.md [hunk 2]
```diff
@@ -287,6 +287,12 @@ The "event publishing" feature publishes each event to the local system event bu
 the @apidoc[ShardingDirectReplication] actor subscribes to these events and forwards them to the replicas allowing them
 to fast forward the stream of events for the origin replica. (With additional potential future support in journals for fast forwarding [#29311](https://github.com/akka/akka/issues/29311)). 
 
+## Hot Standby
+
+If all writes occur to one replica the other replicas are not started there might be many replicated events to catch up with when they are later started. Therefore it can be good to activate all replicaswhen there is some activity. 
+
+This can be achieved automatically when `ReplicatedSharding` is used and direct replication of events is enabled as described in @ref[Direct Replication of Events](#direct-replication-of-events). When each written event is forwarded to the other replicas it will trigger them to start if they are not already started.
```
Comment (chbatey) [inline akka-docs/src/main/paradox/typed/replicated-eventsourcing.md:294]: Will it? I think that was the case with how multi dc worked but we don't send via sharding

Comment (johanandren) [reply akka-docs/src/main/paradox/typed/replicated-eventsourcing.md:294]: We will send it via sharding when Direct Replication of Events is enabled, it's a local actor on each nodes which subscribes to the local event bus and sends the direct-replica-events that is published over sharding to the other replicas (which will wake them up)

Comment (chbatey) [reply akka-docs/src/main/paradox/typed/replicated-eventsourcing.md:294]: ofc, brain was not engaged

**Review by chbatey — APPROVED** [review]

**Pull request merged**